cypress/integration/app.spec.js

```diff
@@ -0,0 +1,202 @@
+import { CTOS } from '../../src/core/enums';
+
+function logToWindow(log) {
+  cy.window().then(win => win.console.log('[CYPRESS]', log))
+}
+
+context('App', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:80');
+
+    cy.window().then((window) => {
+      window.localStorage.setItem(
+        'imageURL',
+        'https://ygoprodeck.com/pics/'
+      );
+    });
+
+    // cy.get('#openlogin').click();
```

Comment:
forgot to uncomment this, but this is needed if the initial state of the login component is hard coded to `loading` instead of `login`